manager/orchestrator/updater.go

```diff
@@ -275,6 +275,7 @@ func (u *Updater) updateTask(ctx context.Context, instance []*api.Task, updated
 	})
 	defer cancel()
 
+	stopAfterStart := false
```

Comment:
The option should be called startThenStop. I'm thinking `startFirst` might also be a good name. In that case the default is `stopFirst`.